code/src/components/Product.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+export const Product = () => {
+  const scan = useSelector(state => state.products.product);
+
+  if (!scan) return null;
+
+  console.log("SCAN", scan);
+
+  return (
+    <>
+      {scan.product && scan.status === 1 && (
+        <section className="scanner-container">
+          <img
+            alt="barcode"
+            src={scan.product && scan.product.image_nutrition_url}
+          />
```

Comment:
To make this even nicer you could add a default image as a backup. Since there are many products without the image_nutriotion_url data.